packages/makeswift/package.json

```diff
@@ -33,7 +33,7 @@
     "@babel/traverse": "^7.18.13",
     "chalk": "^4.1.2",
     "commander": "^9.4.0",
-    "create-next-app": "^12.2.5",
+    "create-next-app": "^12.3.1",
```

Comment:
Should we make this `*` so that we don't have to upgrade all the time? Or would you rather we keep a stable version? we could also do `12` so that it automatically gets minor versions in addition to patch

Reply:
I suppose we should stick within a major version. But yes I don't want to be upgrading all the time

Reply:
Is that `12.x`?

I will also say — I don't care that much if we're out of date by a few minor versions. I think it's about whether or not the CLI works, not if that warning is in the output.

Reply:
Hmm ok. If you don't care about the warning, then the caret version should be enough. We could also suppress the warning

Reply:
if the caret version is enough, why did the previous caret version get a warning? I guess the lock file?

Reply:
Because of caches/lockfiles. When you run `npx` it installs a version and caches it. Next time you run it it will use that cached version and the installed version could be outdated. If you do `npx makeswift@latest init` then the warning shouldn't be there